src/duplicacy_backupmanager.go

```diff
@@ -1048,7 +1054,7 @@ func (manager *BackupManager) UploadSnapshot(chunkMaker *ChunkMaker, uploader *C
                 if _, found := chunkCache[chunkID]; found {
                     completionFunc(chunk, 0, true, chunk.GetLength(), 0)
                 } else {
-                    uploader.StartChunk(chunk, len(sequence))
+                     uploader.StartChunk(chunk, len(sequence))
```

Comment:
What's this doing here? It looks like a formatting-only change that makes it not line up with line 1055.